api/TextTrackCue.json

```diff
@@ -78,10 +78,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "6.1"
+              "version_added": "6"
             },
             "safari_ios": {
-              "version_added": "8"
+              "version_added": "7"
```

Comment:
Is this based on mirroring? Wasn't a bunch of text track stuff enabled in iOS 8?

Reply:
This was based on testing, actually.  The text track APIs were enabled in iOS 7 rather than iOS 8.

Reply:
Alright then!